draft-ietf-masque-h3-datagram.md

```diff
@@ -731,7 +839,7 @@ DATAGRAM                       -------->
 STREAM(44): CAPSULE             -------->
   Capsule Type = REGISTER_DATAGRAM_CONTEXT
   Context ID = 2
-  Extension String = "ip=192.0.2.42,port=443"
+  Context Extension = {IP_COMPRESSION=192.0.2.42:443}
```

Comment:
I find this example kind of confusing, especially now that contexts are bidirectional.  A more realistic hypothetical might be something like `IP_COMPRESSION=tcp,192.0.2.41:54321,192.0.2.42:443`.  Or just `IP_COMPRESSION=<TBD connection info>`.